cms-contentful/src/main/java/io/commercetools/sunrise/cms/contentful/ContentfulCmsService.java

```diff
@@ -1,17 +1,95 @@
 package io.commercetools.sunrise.cms.contentful;
 
+import com.contentful.java.cda.CDAArray;
+import com.contentful.java.cda.CDAAsset;
+import com.contentful.java.cda.CDAClient;
+import com.contentful.java.cda.CDAEntry;
 import io.commercetools.sunrise.cms.CmsIdentifier;
 import io.commercetools.sunrise.cms.CmsService;
 
+import javax.annotation.Nullable;
 import java.util.List;
 import java.util.Locale;
+import java.util.Map;
 import java.util.Optional;
+import java.util.Set;
+import java.util.concurrent.CompletableFuture;
 import java.util.concurrent.CompletionStage;
 
 public class ContentfulCmsService implements CmsService {
 
+    private static final String ENTRY_TYPE = "content_type";
+    private static final String ENTRY_KEY = "fields.name[match]";
+    private static final String LIMIT = "limit";
+    private static final String ONE = "1";
```

Comment:
1 is not considered a magic number, but if you want to keep it as a constant, which is perfectly fine for me, I would give it a meaningful name, otherwise you could use the same `ONE` for everything that you need 1, and that would be killing the purpose. As it's your limit, then `LIMIT` could be an option. In the case of the previous constant `CDA_LIMIT_KEY` could be an option, since it's actually not the limit, but the name that CDA uses for limit.